sentry-jul/src/main/java/io/sentry/jul/SentryHandler.java

```diff
@@ -0,0 +1,283 @@
+      }
+    } catch (RuntimeException e) {
+      reportError(
+          "An exception occurred while creating a new event in Sentry",
+          e,
+          ErrorManager.WRITE_FAILURE);
+    }
+  }
+
+  /** Retrieves the properties of the logger. */
+  private void retrieveProperties() {
+    final LogManager manager = LogManager.getLogManager();
+    final String className = SentryHandler.class.getName();
+    setPrintfStyle(Boolean.parseBoolean(manager.getProperty(className + ".printfStyle")));
+    setLevel(parseLevelOrDefault(manager.getProperty(className + ".level")));
+    final String minimumBreadCrumbLevel =
+        manager.getProperty(className + ".minimumBreadcrumbLevel");
+    if (minimumBreadCrumbLevel != null) {
+      setMinimumBreadcrumbLevel(parseLevelOrDefault(minimumBreadCrumbLevel));
+    }
+    final String minimumEventLevel = manager.getProperty(className + ".minimumEventLevel");
+    if (minimumEventLevel != null) {
+      setMinimumEventLevel(parseLevelOrDefault(minimumEventLevel));
+    }
+  }
+
+  /**
+   * Transforms a {@link Level} into an {@link SentryLevel}.
+   *
+   * @param level original level as defined in JUL.
+   * @return log level used within sentry.
+   */
+  private static @Nullable SentryLevel formatLevel(final @NotNull Level level) {
+    if (level.intValue() >= Level.SEVERE.intValue()) {
+      return SentryLevel.ERROR;
+    } else if (level.intValue() >= Level.WARNING.intValue()) {
+      return SentryLevel.WARNING;
+    } else if (level.intValue() >= Level.INFO.intValue()) {
+      return SentryLevel.INFO;
+    } else if (level.intValue() >= Level.ALL.intValue()) {
+      return SentryLevel.DEBUG;
+    } else {
+      return null;
+    }
+  }
+
+  private @NotNull Level parseLevelOrDefault(final @NotNull String levelName) {
+    try {
+      return Level.parse(levelName.trim());
+    } catch (RuntimeException e) {
+      return Level.WARNING;
+    }
+  }
+
+  private @NotNull Breadcrumb createBreadcrumb(final @NotNull LogRecord record) {
+    final Breadcrumb breadcrumb = new Breadcrumb();
+    breadcrumb.setLevel(formatLevel(record.getLevel()));
+    breadcrumb.setCategory(record.getLoggerName());
+    if (record.getParameters() != null) {
+      try {
+        breadcrumb.setMessage(formatMessage(record.getMessage(), record.getParameters()));
+      } catch (RuntimeException e) {
+        breadcrumb.setMessage(record.getMessage());
+      }
+    } else {
+      breadcrumb.setMessage(record.getMessage());
+    }
+    return breadcrumb;
+  }
+
+  /**
+   * Creates {@link SentryEvent} from JUL's {@link LogRecord}.
+   *
+   * @param record the log record
+   * @return the sentry event
+   */
+  // for the Android compatibility we must use old Java Date class
+  @SuppressWarnings("JdkObsolete")
+  @NotNull
+  SentryEvent createEvent(final @NotNull LogRecord record) {
+    final SentryEvent event = new SentryEvent(new Date(record.getMillis()));
```

Comment:
`record.getMillis()` is it UTC or local time?
otherwise we need to use `DateUtils.getDateTime(millis)`

Reply:
ok theres a test for it, please ignore